### ISSUE TITLE
Graph line doesn't show on Hololens 1 recording

### ISSUE THREAD

**Issue by DanCRichards** [cre8ivepark/COVID19DataVisualizationHoloLens2#4]
**Steps to reproduce**
1. Run the program and ensure line data can be shown
2. Use the hololens screen capture tool to record. 
3. Playback recording

I don't like creating issues on someone else's project, but I hope this helps with you knowing :) 
![image](https://user-images.githubusercontent.com/17732740/78216199-c8b09480-7515-11ea-8b42-f2cb1cc4f9f8.png)
![image](https://user-images.githubusercontent.com/17732740/78216320-0e6d5d00-7516-11ea-8048-73962eb5b82b.png)

**Comment by cre8ivepark** [comment]
@DanCRichards Thanks! I noticed the same issue when I was testing. I found that it was rendering only on one eye. Need to configure to 'Single Pass' instead of 'Single Pass Instanced'. I don't experience that much of performance issue with this change.
![2020-04-02 21_46_58-Unity 2018 4 12f1 Personal - COVID19HoloLens2 unity - COVIDDataVizGit - Universa](https://user-images.githubusercontent.com/13754172/78325135-a32e8580-752b-11ea-8aa6-19f1d29cf5eb.png)
